roles/repositories/tasks/main.yml

```diff
@@ -9,7 +9,7 @@
     product: "{{ item.0.name }}"
     name: "{{ item.1.name }}"
     all_repositories: false
-    state: enabled
+    state: "{{ item.1.state | default(omit) }}"
```

Comment:
I decided to `default(omit)` here (and later) as `enabled` and `present` are the defaults anyway.